l10n_cl_invoicing_policy/models/res_partner.py

```diff
@@ -23,19 +23,21 @@ def _default_invoicing_policy(self):
         default=_default_invoicing_policy,
         help="""
         * Ticket: Only for individuals. 1 invoice for each delivery order.
-        * Invoice: Only for companies. The VAT is required. 1 invoice for each
-          delivery order. Requires the customer PO # on the SO/Invoice.
-        * Electronic Guide: Only for companies. 1 invoice at the end of the
-          month for all the delivery orders of that month.""")
+        * Invoice: Only for companies and their children. The VAT is required.
+          1 invoice for each delivery order.
+          Requires the customer PO # on the SO/Invoice.
+        * Electronic Guide: Only for companies and their children.
+          1 invoice at the end of the month for all the delivery orders of
+          that month.""")
 
-    @api.constrains('is_company', 'invoicing_policy')
+    @api.constrains('is_company', 'invoicing_policy', 'parent_id')
     def check_invoicing_policy(self):
-        if self.invoicing_policy == 'ticket':
-            if self.is_company:
+        if self.is_company or self.parent_id:
```

Comment:
This is incorrect: it doesn't allow for child Addresses of individuals to be set to `ticket`.
See #97 for an implementation suggestion.

Reply:
@dreispt Yes, this is the desired behavior... Right @nelsonramirezs?